diff --git a/dist/vue-i18n.esm-bundler.js b/dist/vue-i18n.esm-bundler.js
index 7dc04f5024352d5900c70cd9b58a12dfe74400d3..076959735fe10f663ea4a5249534699a055610b2 100644
--- a/dist/vue-i18n.esm-bundler.js
+++ b/dist/vue-i18n.esm-bundler.js
@@ -23,30 +23,18 @@ const VERSION = '9.1.9';
  * istanbul-ignore-next
  */
 function initFeatureFlags() {
-    let needWarn = false;
     if (typeof __VUE_I18N_FULL_INSTALL__ !== 'boolean') {
-        needWarn = true;
         getGlobalThis().__VUE_I18N_FULL_INSTALL__ = true;
     }
     if (typeof __VUE_I18N_LEGACY_API__ !== 'boolean') {
-        needWarn = true;
         getGlobalThis().__VUE_I18N_LEGACY_API__ = true;
     }
     if (typeof __VUE_I18N_PROD_DEVTOOLS__ !== 'boolean') {
-        needWarn = true;
         getGlobalThis().__VUE_I18N_PROD_DEVTOOLS__ = false;
     }
     if (typeof __INTLIFY_PROD_DEVTOOLS__ !== 'boolean') {
         getGlobalThis().__INTLIFY_PROD_DEVTOOLS__ = false;
     }
-    if ((process.env.NODE_ENV !== 'production') && typeof true === 'boolean') {
-        needWarn = true;
-    }
-    if ((process.env.NODE_ENV !== 'production') && needWarn) {
-        console.warn(`You are running the esm-bundler build of vue-i18n. It is recommended to ` +
-            `configure your bundler to explicitly replace feature flag globals ` +
-            `with boolean literals to get proper tree-shaking in the final bundle.`);
-    }
 }
 
 const warnMessages = {
